ITI/TF/Volume1/ch-10.html

```diff
@@ -747,6 +753,13 @@ <h3 id="10.2.10">10.2.10 Delayed Document Assembly Option</h3>
       </ul>
       <p>See <a href="../Volume2/ITI-42.html#3.42.4.1.2.1">ITI TF-2:3.42.4.1.2.1</a> and <a href="../Volume2/ITI-43.html#3.43.4.1.3.2">ITI TF-2:3.43.4.1.3.2</a></p>
       <p>In order for an Integrated Document Source/Repository to support the Delayed Document Assembly Option it must interact with a Document Registry supporting the Document Metadata Update Option.</p>
+      <h3 id="10.2.11">10.2.11 Target Communities Option</h3>
+      <p>The Document Consumer which supports the Target Communities Option:</p>
+      <ul>
+         <li class="bullet-list0">In Registry Stored Queries which require a patient identifier, can specify the “$targetCommunityIdList” parameter, containing the list of the communities to which it wants to direct the request to.</li>
+	 <li class="bullet-list0">In queries in which the “$targetCommunityIdList” is specified, the Document Consumer shall not use the “$home” parameter.</li>
+      </ul>
+      <p>Note that when addressing the query to the Initiating Gateway, the Document Consumer that specifies the “$targetCommunityIdList” parameter must be aware that the Initiating Gateway may not support the “Target Communities” Option. See Section 3.18.4.1.2.5 Compatibility of Options.</p>
```

Comment:
i suggest to put a reference on see section 3.18.4.1.2.5

e.g. See Section [ITI TF-2:3.18.4.1.2.5](https://github.com/IHE/publications/pull/Volume2/ITI-18.html#3.18.4.1.2.5) Compatibility of Options.